packages/components/select-v2/src/select.ts

```diff
@@ -0,0 +1,14 @@
+import { CHANGE_EVENT, UPDATE_MODEL_EVENT } from '@element-plus/constants'
+import { isBoolean } from '@element-plus/utils'
+
+export const selectV2Emits = {
+  [UPDATE_MODEL_EVENT]: (value: any) => value || true,
+  [CHANGE_EVENT]: (value: any) => value || true,
+  'remove-tag': (tagValue: any) => tagValue || true,
+  clear: () => true,
+  'visible-change': (visible: boolean) => isBoolean(visible),
+  focus: (evt?: FocusEvent) => evt instanceof FocusEvent || evt === undefined,
+  blur: (evt?: FocusEvent) => evt instanceof FocusEvent || evt === undefined,
+}
+
+export type SelectV2Emits = typeof selectV2Emits
```

Comment:
Thank for your contribution.

Suggest moving it to the `defaults.ts` file.

Avoid using `any` and try to perfect the type as much as possible. Reference ElInput

Reply:
Thank you for your review! 👍 

I will look into it and try making some changes in a few days.